fiftyone/utils/ptv3_arch/serialization/hilbert.py

```
@@ -0,0 +1,294 @@
+    Returns:
+    --------
+     The output is an ndarray of unsigned integers with the same shape as hilberts
+     but with an additional dimension of size num_dims.
+    """
+
+    if num_dims * num_bits > 64:
+        raise ValueError("""
+      num_dims=%d and num_bits=%d for %d bits total, which can't be encoded
+      into a uint64.  Are you sure you need that many points on your Hilbert
+      curve?
+      """ % (num_dims, num_bits))
```

Ruff (0.15.17)

[error] `%`-format string has 3 placeholder(s) but 2 substitution(s)

(F507)